pkg-scripts/vircadia-assignment-client.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=Assignment client service for Vircadia server
+Description=Assignment client service for the Vircadia Server
```

Comment:
I can't add it as a suggestion in GitHub here, but why didn't you make the change in the other file as well?

Reply:
Probably because I missed it! I'll fix it.